### PR TITLE
Graphql tooling 

### PR DESCRIPTION
**Pull request by danielwilms**
Upgrade of graphql as community supported tool
    
    - adopt graphql to use exporters
    - remove additional fields `source` and `channel`
    - add graphql specific parameter to add additional metadata
    - adopt readme
    - remove graphql exporters from contrib
    
    call with:
    ```
    ${TOOLSDIR}/vspec2graphql.py -I ./spec --gqlfield "source" "Source System" --gqlfield "channel" "Collecting channel" ./spec/VehicleSignalSpecification.vspec vss_rel_$$(cat VERSION).graphql.ts
    ```

### CODE REVIEW

File: vspec2graphql.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+#
+#
+# (c) 2022 BMW Group
+#
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+#
+# Convert vspec2json wrapper for vspec2x
```
Comment (erikbosch) [inline vspec2graphql.py:10]: Typo, not vspec2json

**Comment by danielwilms** [comment]
@danielwilms rebase and fix comments, then ready to merge

Decision to merge 2022/06/07

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Examples on tool usage can be found in the [VSS Makefile](https://github.com/COV
 [vspec2ocf.py](contrib/ocf/vspec2ocf.py) | Parses and expands a VSS and generates a OCF specification | Contrib (Obsolete, no longer functional)  | - |
 [vspec2proto.py](contrib/vspec2protobuf.py) | Parses and expands a VSS and generates a Protobuf specification | Contrib (Beta)  | - |
 [vspec2ttl.py](contrib/vspec2ttl/vspec2ttl.py) | Parses and expands a VSS and generates a TTL specification | Contrib  | - |
-[vspec2graphql.py](contrib/vspec2graphql.py) | Parses and expands a VSS and generates a GraphQL specification | Contrib  | - |
+[vspec2graphql.py](contrib/vspec2graphql.py) | Parses and expands a VSS and generates a GraphQL specification | Community Supported  | - |
```
[the remark this answers is not in the document]
Comment (danielwilms) [reply README.md]: done.

**Comment by danielwilms** [comment]
@erikbosch, rebased, fixed typo and added the link to the documentation

**Comment by danielwilms** [comment]
@erikbosch, please wait with merging. found one bug still

**Comment by danielwilms** [comment]
> @erikbosch, please wait with merging. found one bug still

Should be fine now...please run it once on your machine (you just have to remove `contrib/` from the `graphql` target in the Makefile)

**Comment by erikbosch** [comment]
Did a sanity check and it worked fine. Merging it. I assume next step for you is to update Makefile in VSS and update submodule

**Pull request merged**